XLS-0070d-credentials/README.md

```diff
@@ -349,8 +332,8 @@ Note: the transaction will still fail if too many credentials are included. The
 
 ### 8.3. State Changes
 
-* If the credential isn't valid, the transaction fails.
-* If the credential is expired, the credential is deleted.
+* If a credential isn't valid, the transaction fails.
+* If a credential is expired, the credential is deleted.
```

Comment:
and transaction fails

Reply:
The credential is deleted regardless of whether the transaction succeeds or fails, right?

Reply:
The transaction will always fails if credential(s) are expired, and expired credentials will be deleted